generated_golden.sh

```diff
@@ -79,6 +79,13 @@ scaffold_test_project() {
 }
 
 set -e
+
+if ! git lfs > /dev/null 2>&1 ; then
+    echo "this project requires git-lfs, see: https://git-lfs.github.com/"
+    echo "after installing, you'll need to git checkout ./testdata/*tgz"
+    exit 1
+fi
+
 build_kb
-scaffold_test_project gopath/src/project 1
+#scaffold_test_project gopath/src/project 1
```

Comment:
uncomment :)

Reply:
Oops, thought I caught that one. Will fix.